clientd3d/config.c

```diff
@@ -294,9 +308,25 @@ void ConfigLoad(void)
    config.active_stat_group = GetConfigInt(misc_section, INIActiveStatGroup, 5, ini_file);
 
    // Determine if we should be using gpu efficiency mode or not.
-   char config_value[255];
-   GetPrivateProfileString("config", "gpuefficiency", "error", config_value, 255, "./config.ini");
-   config.gpuEfficiency = (0 == strcmp(config_value, "true"));
+   auto one_time_gpu_efficiency_enablement = GetConfigInt(config_section, 
+	   "gpuefficiencyonetimeflip", False, config_ini_file);
+   if (one_time_gpu_efficiency_enablement)
+   {
+      char config_value[10];
+      GetPrivateProfileString(config_section, INIGpuEfficiency, "true", config_value, 
+		  sizeof(config_value), config_ini_file);
+      config.gpuEfficiency = (0 == strcmp(config_value, "true"));
+   }
+   else
+   {
+      // TODO: Added January 2025 - remove after the next update when everyone will have been reset.
+      // Perform one-time flip to enable GPU efficiency mode regardless of current preference.
+      // This is to ensure players start from a clean slate with maximum performance.
+      // After this one-time flip, future changes to gpu efficiency preferences will be respected.
```

Comment:
Should one_time_gpu_efficiency_enablement also be removed at that time?  Make it easy on the future maintainer (who could be you).